lib/vsc/utils/mail.py

```diff
@@ -85,13 +103,20 @@ def _send(self,
         """
 
         try:
-            if self.mail_host:
-                logging.debug("Using %s as the mail host", self.mail_host)
-                s = smtplib.SMTP(self.mail_host)
+            if self.auth:
+                logging.debug("Using %s as the mail host with authenticatio", self.mail_host)
+                s = smtplib.SMTP(host=self.mail_host, port=self.mail_port)
+                context = ssl.create_default_context()
+                s.starttls(context=context)
+                s.login(user=self.go.options.smtp_auth_user, password=self.go.options.smtp_auth_password)
             else:
-                logging.debug("Using the default mail host")
-                s = smtplib.SMTP()
-                s.connect()
+                if self.mail_host:
+                    logging.debug("Using %s as the mail host", self.mail_host)
+                    s = smtplib.SMTP(self.mail_host)
+                else:
+                    logging.debug("Using the default mail host")
+                    s = smtplib.SMTP()
+                    s.connect()
```

Comment:
is the indent correct here? no connect if mail_host is configured?

Reply:
It was like this forever, but I'm not sure now.